server/src/main/java/org/mixit/cesar/cfp/model/ProposalVoteValue.java

```diff
@@ -0,0 +1,12 @@
+package org.mixit.cesar.cfp.model;
+
+/**
+ * @author mpetitdant
+ *         Date: 03/02/16
+ */
+public enum ProposalVoteValue {
+    NEED_IT,
+    GOOD,
+    BAD,
+    VETO
+}
```

Comment:
You can store a value with each value -2, -1,... to be able to simplify the future computation